app/javascript/v3/api/auth.js

```diff
@@ -42,6 +42,19 @@ export const register = async creds => {
   return null;
 };
 
+export const accountSetup = async ({ id, name, locale }) => {
+  try {
+    const response = await wootAPI.put(`api/v2/accounts/${id}.json`, {
+      name,
+      locale,
+    });
+    return response.data;
+  } catch (error) {
+    throwErrorMessage(error);
+  }
+  return null;
```

Comment:
I guess this statement would never be executed.

Reply:
Makes sense. Removed it.